source/user-manual/indexer-api/getting-started.rst

```diff
@@ -0,0 +1,870 @@
++--------------------------------------+------------------------------------------+-------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``required_issuer``                  | Any string                               | null              | Sets the target issuer of a JWT stored in the JSON payload.                                                                                                                                                                                                               |
++--------------------------------------+------------------------------------------+-------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``jwt_clock_skew_tolerance_seconds`` | Any positive integer                     | 30                | Sets a time window (in seconds) to compensate for any disparity between the JWT authentication server and Wazuh indexer node clock times. It prevents authentication failures due to misalignment.                                                                        |
++--------------------------------------+------------------------------------------+-------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+jwt_auth_domain.authentication_backend
+""""""""""""""""""""""""""""""""""""""
++----------------------+---------------------+--------------------+------------------------------------------------------------------------------------------------------------------------------------------------+
+| **Sub-fields**       | **Allowed values**  | **Default value**  | **Description**                                                                                                                                |
++======================+=====================+====================+================================================================================================================================================+
+| ``type``             | noop                | noop               | This value is set to no operation (noop) because JWTs are self-contained and the user is authenticated at the HTTP level.                      |
```

Comment:
```suggestion
| ``type``             | noop                | noop               | This value is set to no operation (``noop``) because JWTs are self-contained and the user is authenticated at the HTTP level.                  |
```